src/Elasticsearch/State/ItemProvider.php

```diff
@@ -49,9 +49,9 @@ public function __construct(
     public function provide(Operation $operation, array $uriVariables = [], array $context = []): ?object
     {
         $resourceClass = $operation->getClass();
-        $options = $operation->getStateOptions() instanceof Options ? $operation->getStateOptions() : new Options(index: $this->getIndex($operation));
```

Comment:
Here options was always an Options.
- Phpstan knows
- PHPStorm doesn't

So I rewrite the code to help phpstorm understand it and avoid the false positive for phpstan